src/main/app/src/app/shared/material-form-builder/form-components/file/file.component.ts

```diff
@@ -149,7 +149,7 @@ export class FileComponent extends FormComponent implements OnInit {
     return super.getErrorMessage();
   }
 
-  droppedFile(file: File): void {
-    this.uploadFile(file);
+  droppedFile(files: FileList): void {
+    files.length && this.uploadFile(files[0]);
```

Comment:
Should multiple files not be rejected? 
If only the first one is handled anyway, it should be clear which one that is, therefore only allow a single file to be dropped?

Reply:
I'm not sure. It might be nice if we can signal to the user somehow that they can only drop a single file, but I don't think that's part of this story. You can see the file name, so I guess the user knows which file will be uploaded.